src/main/java/uk/gov/hmcts/darts/arm/service/impl/UnstructuredToArmBatchProcessorImpl.java

```diff
@@ -45,7 +39,7 @@
 public class UnstructuredToArmBatchProcessorImpl implements UnstructuredToArmBatchProcessor {
```

Comment:
We now have 2 batch push processors:
- `UnstructuredToArmBatchProcessor`
- `DetsToArmBatchPushProcessor`
I think it would be useful if they implemented a common top level `ArmBatchProcessor` "tag" interface. 
It's getting more and more difficult to keep track of all these Arm related processors and their multiple variants

Reply:
yea its work optimising it in future but not part of this ticekt.